packages/flame_3d/lib/src/components/group_3d.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flame_3d/components.dart';
+
+/// {@template group_3d}
+/// [Group3D] is a [Component3D] that acts as a container for other
+/// [Component3D]s.
+///
+/// Changing the position, rotating or scaling the [Component3D] will affect
+/// the whole group as if it was a single entity.
```

Comment:
I'm not convinced this will "just work", neither before nor now. But that is a test for a followup.

Reply:
But the class itself would not be needed anyway because when it works the `Component3D` does the same thing :P

Reply:
I still don't think we need the class tho, grouping is a core feature of any component.

Reply:
removed